packages/metrics/src/index.ts

```diff
@@ -62,27 +66,38 @@ class Metrics {
   }
 
   bootstrapSentry() {
-    const { context, ignoreErrors, sentryKey, sentryProject, userID } = this.settings;
-
-    if (!sentryKey || !sentryProject) {
+    const {
+      context,
+      ignoreErrors,
+      sentryDSN,
+      sentryKey,
+      sentryProject,
+      userID,
+      onSentryScope,
+    } = this.settings;
+    const { host, protocol } = global.location;
+
+    if (!sentryDSN && (!sentryKey || !sentryProject)) {
       return;
     }
 
-    const ravenHost = `${global.location.protocol}//${sentryKey}@${global.location.host}`;
-    const ravenPath = `/proxy/sentry/${sentryProject}`;
-
-    Raven.config(`${ravenHost}${ravenPath}`, {
+    initSentry({
+      dsn: sentryDSN || `${protocol}//${sentryKey}@${host}/${sentryProject}`,
```

Comment:
dsn accepts a breakdown that can make this a bit easier.
https://git.musta.ch/airbnb/torch/blob/master/src/setup/setupMetrics.js#L28